src/components/TextInput/TextInput.scss

```diff
@@ -0,0 +1,112 @@
+@use '~@quartz/styles/scss/borders';
+@use '~@quartz/styles/scss/color-scheme';
+@use '~@quartz/styles/scss/fonts';
+@use '~@quartz/styles/scss/helpers/resets';
+@use '~@quartz/styles/scss/tokens';
+
+.container {
+	position: relative;
+	border: 0;
+	padding: 0;
+	margin: 0;
+}
+
+.label {
+	@include fonts.maison-extended-700-1;
+
+	text-transform: uppercase;
+	color: color-scheme.$typography;
+	display: block;
+	line-height: 1.25;
+
+	&.required {
+		display: flex;
+		flex-wrap: wrap;
```

Comment:
doesn't seem like you should need flexbox here, e.g. could you add a space before the asterisk? `content: ' *'`

Reply:
Hmm it was either this or switching the `.label` class back to an inline-block element and futzing with the vertical alignment of the pseudo-element since it's a different font size - this seemed a little cleaner